templates/endpoints_v2/README.ipynb

```diff
@@ -6,37 +6,54 @@
    "source": [
     "# Deploy, configure, and serve LLMs \n",
     "\n",
-    "**⏱️ Time to complete**: 10 min (20 on GCP)\n",
+    "**⏱️ Time to complete**: 10 min\n",
     "\n",
-    "This guide walks you through how to deploy optimized LLMs in Anyscale through RayLLM. It includes a number of pre-tuned configs for Llama2, Mistral, Mixtral, embedding models, and more in the `models` directory.\n",
+    "This template comes with a library for serving OSS LLMs on Anyscale called [RayLLM](http://https://docs.anyscale.com/llms/serving/intro).\n",
     "\n",
-    "You can also find more advanced tutorials in the `cookbooks/` folder, including those for:\n",
-    "- Embedding generation\n",
-    "- Deploying custom models\n",
-    "- Deploying LoRA and function-calling models\n",
-    "- Deploying vision language models\n",
-    "- How to configure autoscaling and other optimization parameters\n",
+    "RayLLM provides a number of features that simplify LLM development, including:\n",
+    "- An extensive suite of pre-configured open source LLMs.\n",
+    "- An OpenAI-compatible REST API.\n",
     "\n",
-    "**Note**: This guide is hosted within an Anyscale workspace, which provides easy access to compute resources. Check out the `Introduction to Workspaces` template for more details.\n",
+    "As well as operational features to efficiently scale LLM apps:\n",
+    "- Optimizations such as continuous batching, quantization and streaming.\n",
+    "- Production-grade autoscaling support, including scale-to-zero.\n",
+    "- Native multi-GPU & multi-node model deployments.\n",
     "\n",
-    "## Step 1 - Run the model locally in the Workspace\n",
+    "This template explains how to set up, run, and query LLMs with RayLLM.\n",
     "\n",
-    "We provide a starter command to run Llama and Mistral-family models via Ray Serve. You can specify the arguments, such as Lora, GPU type and tensor parallelism via the command. You can also follow the [guide](cookbooks/CustomModels.ipynb) to bring your own models.\n",
+    "To learn more about RayLLM, check out [the docs](http://https://docs.anyscale.com/llms/serving/intro). If you're interested in fine-tuning, check out the [fine-tuning template](../fine-tune-llm_v2/README.ipynb).\n",
```

Comment:
link the actual workspace template for [fine-tuning template] i.e. https://console.anyscale.com/v2/template-preview/finetuning_llms_v2

Reply:
Got it, I updated the template links.